paper-password-field.html

```diff
@@ -47,19 +47,12 @@
   <template>
     <style include="paper-text-field-shared-styles"></style>
 
-    <div class="floated-label-placeholder" hidden$="[[noFloatLabel]]">&nbsp;</div>
-
-    <div class="input-content">
+    <div class="container">
```

Comment:
It's not being used in a `this.$.container` way, so it doesn't need an id (but needs to be styled)